test/components/redButton.spec.js

```diff
@@ -17,8 +19,11 @@ describe('RedButton', () => {
     expect(component.tagName).toBe('BUTTON');
   });
 
-  it('Created button should have redButton class', () => {
-    const component = RedButton('HELLO');
-    expect(component.classList.contains('redButton')).toBe(true);
+  it('When button is clicked then console.log message should be displayed', () => {
```

Comment:
Super, że udało Ci się napisać ten test :) Teraz mamy w kodzie dowód, że button działa i jak ktoś go popsuje będziemy wiedzieć kogo ścigać.

Można by to uprościć do:
```
    const testFunction = jest.fn()
    const component = RedButton('someText', testFunction);
    userEvent.click(component);
    expect(testFunction).toBeCalled();
```